index.html

```diff
@@ -57,25 +57,238 @@ <h2>you!</h2>
+                <p>Hello I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="about-myself-socialmedia">
+                <div class="about-myself-letsconnect">
+                    <h2>LET'S CONNECT</h2>
+                </div>
+                <div class="about-myself-sociallinks">
+                    <ul class="about-myself-sociallists">
+                        <li> <a href="https://twitter.com/DannyDotcoder" target="_blank" rel="noreferrer noopener">
+                                <img src="img/Vector.png" alt="vector">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/daniel-shdeed-832b03115/" target="_blank"
+                                rel="noreferrer noopener">
+                                <img src="img/Linkedinicon.png" alt="linkedinicon">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" target="_blank" rel="noreferrer noopener">
+                                <img src="img/Vector(1).png" alt="vector1">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://github.com/Danieldotcomcoder" target="_blank" rel="noreferrer noopener">
+                                <img src="img/Vector(2).png" alt="vector2">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" target="_blank" rel="noreferrer noopener">
+                                <img src="img/Vector(3).png" alt="vector3">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+
+            </div>
+            <div class="about-myself-getmyresume">
+                <a href="#" rel="noreferrer noopener">Get my resume</a>
+            </div>
         </div>
-    </div>
+        <div class="skill-list-section">
+            <ul class="skill-list-categories">
+
+                <li class="skill-languages-list">
+                    <div class="skill-language-text">
+                        <a href="#" rel="noreferrer noopener">
+                            <h2>Languages</h2>
+                            <img src="img/Union1.png" alt="union1">
+                        </a>
+                    </div>
+                    <a href="#" rel="noreferrer noopener"><img src="img/Frame9.png" alt="frame9"></a>
+                    <a href="#" rel="noreferrer noopener"><img src="img/Frame10.png" alt="frame10"></a>
+                    <a href="#" rel="noreferrer noopener"><img src="img/Frame11.png" alt="frame11"></a>
```

Comment:
- [ ] Please consider moving these `a` tags inside `li` tags that are inside another `ul` tag. Something like this:
```
<ul>
   <li><a href=>Link</a></li>
</ul>
```
This will make your code more semantic. 👍🏻